README.md

```diff
@@ -34,51 +35,63 @@ git add HistFitter && git commit -m "Add HistFitter submodule"
 
 Recommended Root version is 6.28/00. This version of HistFitter is not compatible with ROOT versions < 6.28.
 
-## Setup
-
-The environment can be set doing
-
-```
-source setup.sh
-```
-
 An LCG release with the correct ROOT and Python versions can be loaded with
 
 ```
 source setup_lcg.sh
```

Comment:
This works for the ATLAS environment and not outside of it.  I would remove this from the README.  People will need to setup ROOT in various ways, and ATLAS specific suggestions can go onto the internal twiki.

Reply:
Do you want to remove the reference to LCG release? It is in the current README.

Reply:
Hmm, yes I missed that.  Ok, you can leave it and I'll fix it later.